protocols/api/src/main/java/org/apache/james/protocols/api/logger/ContextualLogger.java

```diff
@@ -42,7 +44,7 @@ public String get() {
         }, session.getSessionID(), logger);
     }
 
-    public ContextualLogger(Supplier<String> userSupplier, String sessionId, Logger logger) {
+    public ContextualLogger(Supplier<String> userSupplier, String sessionId, org.slf4j.Logger logger) {
```

Comment:
Ok, so the full concept of ContextualLogger is no longer relevant : we should use MDC for that (see slf4j and/or logback MDC for more information).
As MDC are ThreadLocal, we have to solve thread + MDC in our context.